workshops/templates/workshops/person_bulk_add_results.html

```diff
@@ -25,13 +25,13 @@
                 <input type="text" name="personal" value="{{ entry.personal|default:"" }}" class="hidden">
             </td>
             <td class="editable">
-                <span>{{ entry.middle|default:"&mdash;" }}</span>
-                <input type="text" name="middle" value="{{ entry.middle|default:"" }}" class="hidden">
-            </td>
-            <td class="editable">
                 <span>{{ entry.family|default:"&mdash;" }}</span>
                 <input type="text" name="family" value="{{ entry.family|default:"" }}" class="hidden">
             </td>
+            <td>
+                <span>{{ entry.username|default:"&mdash;" }}</span>
```

Comment:
Right now the field is populated with one of the values:

1. output from `create_username` function if there's no matching person in the database
2. matched person's username